packages/in_app_purchase/android/src/main/java/io/flutter/plugins/inapppurchase/BillingClientFactoryImpl.java

```diff
@@ -14,6 +14,7 @@
   @Override
   public BillingClient createBillingClient(Context context, MethodChannel channel) {
     return BillingClient.newBuilder(context)
+        .enablePendingPurchases()
```

Comment:
We can guard this by asking `BillingClient` on dart to call an API with the same name `enablePendingPurchases ()`, otherwise do an assertion failure. Although we can fail it on dart side, I think we should always call it on the JAVA side.
On the unified layer, we can either do the same, or hide it from the user. WDYT?

Reply:
I'm going to reach out offline to get some second opinions on this.